src/init/init-starter-files.js

```diff
@@ -6,23 +6,10 @@ import asyncOra from '../util/ora-handler';
 
 const d = debug('electron-forge:init:starter-files');
 
-export const copy = (source, target) =>
-  new Promise((resolve, reject) => {
-    d(`copying "${source}" --> "${target}"`);
-    let rd;
-    let wr;
-    const rejectCleanup = (err) => {
-      rd.destroy();
-      wr.end();
-      reject(err);
-    };
-    rd = fs.createReadStream(source);
-    rd.on('error', rejectCleanup);
-    wr = fs.createWriteStream(target);
-    wr.on('error', rejectCleanup);
-    wr.on('finish', resolve);
-    rd.pipe(wr);
-  });
+export const copy = (source, target) => {
```

Comment:
This function is kind of defunct now that it's just a single line, probably can just call `fs.copy` inline.